docs/en/deposit-and-withdraw-bnb.md

```diff
@@ -5,7 +5,7 @@ title: Deposit and Withdraw BNB
 
 ## Purpose
 
-Loom’s PlasmaChain has recently integrated with Binance Chain, which means users can now easily deposit and withdraw BNB between Binance Chain and Loom.
+Loom Network’s Basechain has recently integrated with Binance Chain, which means users can now easily deposit and withdraw BNB between Binance Chain and Basechain.
```

Comment:
`BNB` -> `BNB and other BEP2 tokens`

Reply:
Yeah, this was written before we rolled out support for BEP2 tokens. Thanks for spotting it.